lib/src/errors.dart

```diff
@@ -99,16 +99,16 @@ class MultiError extends Error {
   /// The order of values is preserved, and if any error occurs, the
   /// [MultiError.errors] list will have errors in the corresponding slots,
   /// and `null` for non-errors.
-  Future<List<Object>> wait<T>(Iterable<Future<T>> futures,
-      {void Function(T successResult) cleanUp}) {
-    Completer<List<Object>> completer;
+  Future<List<Object?>> wait<T>(Iterable<Future<T>> futures,
+      {void Function(T successResult)? cleanUp}) {
+    late Completer<List<Object?>> completer;
     var count = 0;
     var hasError = false;
     var completed = 0;
     // Initialized to `new List(count)` when count is known.
     // Filled with values until the first error, then cleared
     // and filled with errors.
-    List<Object> results;
+    late List<Object?> results;
```

Comment:
And again can use a default value of `const <Never>[];`. (It's the perfect stand-in for any list).